titan-web-client/src/layouts/dashboard/components/sidebar/Sidebar.js

```diff
@@ -67,6 +68,21 @@ class Sidebar extends React.Component {
         {/* /> */}
         {/* </SidebarContentGroup> */}
 
+        <WithAcl
+          options={['mod.titan:canAckEventExcuse']}

```

Comment:
Could you add this ACL option to the WCF package so we can add it in the GUI? I can walk you through the process so you know how to do it in the future, or you could start with the docs and let me know if they're lacking anywhere.